cluster/saltbase/salt/docker/docker-defaults

```diff
@@ -1,3 +1,7 @@
 {% set grains_opts = grains.get('docker_opts', '') -%}
-DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false"
+{% set e2e_opts = '' -%}
+{% if pillar.get('e2e_test_environment', '').lower() == 'true' -%}
+{% set e2e_opts = '-s devicemapper' -%}
```

Comment:
I agree with @jsafrane that it should not be a Kubernetes decision point what storage driver is used for Docker.